model/src/w3odatmd.F90

```diff
@@ -323,10 +338,10 @@ MODULE W3ODATMD
       INTEGER, PARAMETER      :: NOGRP = 10
```

Comment:
All module variables need documentation.

If not, then each undocumented variable will generate a documentation warning. The goal is to have warning-free documentation builds, so that new PRs can be checked for documentation completeness. If the code is already full of warnings, that won't work.

So all module variables should be documentat, as NOGE and IDOUT are below.

Reply:
I was not aware of this.  I'll add this to the module template.